src/config/experimental.ts

```diff
@@ -60,6 +61,7 @@ export const defaultConfig: Record<string, ExperimentalValue> = {
   [POINTS_NOTIFICATIONS_TOGGLE]: { settings: true, value: false },
   [DAPP_BROWSER]: { settings: true, value: IS_TEST ? true : false },
   [SWAPS_V2]: { settings: true, value: false },
+  [ETH_REWARDS]: { settings: true, value: true },
```

Comment:
do you want `value` to be `false` here and have it basically controlled via remote config other than when in dev / TF mode